RELEASE_NOTES.md

```diff
@@ -8,6 +8,51 @@ This document contains release notes for the current major release and all patch
 For prior releases, see
 [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.15.0 (2017-04-27)
+
+### Add required_recipe endpoint
+
+See Chef RFC 89 for a fuller description
+
+Add the ability to serve a required recipe file to chef-clients.
+
+/organizations/<orgname>/required_recipe returns 404 for all organizations
```

Comment:
[nit] in Markdown, this comes up as /organizations/<orgname>/required_recipe  
 -- if you wrap it in backticks, it would be ok: `/organizations/<orgname>/required_recipe`, or use `\<`: /organizations/\<orgname\>/required_recipe